packages/core/test/financial-templates/expiring-multiparty/PricelessPositionManager.js

```diff
@@ -1068,8 +1068,7 @@ contract("PricelessPositionManager", function(accounts) {
     assert.equal((await pricelessPositionManager.getCollateral(sponsor)).toString(), toWei("98.99"));
 
     // Test that regular fees accrue after an emergency shutdown is triggered.
-    const shutdown = await financialContractsAdmin.callEmergencyShutdown(pricelessPositionManager.address);
-    truffleAssert.eventNotEmitted(shutdown, "EmergencyShutdown");
```

Comment:
Side note: Anyone know why this `truffleAssert.eventNotEmitted` was added? Seems kind of weird to check for an event not emitting. I think the intention is to make sure that the emergency shutdown executes but this test should just assume it works.